index.js

```diff
@@ -59,13 +86,15 @@ export default class AWSHttpProvider extends HttpProvider {
       signer.addAuthorization(credentials, new Date());
       request.setRequestHeader('Authorization', req.headers['Authorization']);
       request.setRequestHeader('X-Amz-Date', req.headers['X-Amz-Date']);
-      if (process.env.AWS_SESSION_TOKEN) {
-        request.setRequestHeader('X-Amz-Security-Token', process.env.AWS_SESSION_TOKEN);
-      }
```

Comment:
I believe this is still needed to support ephemeral user credentials. Please explain or bring it back.

Reply:
unintentionally removed. This is my error. Reverting this change.